### DISCUSSION TITLE
嗨~能简单解释下，为什么需要打补丁才能开启 DRA 么？

### DISCUSSION THREAD

**Discussion by fuhaiq** [apache/celeborn#787] [Q&A] [answered]
嗨~能简单解释下，为什么需要打补丁才能开启 DRA 么？[Spark3 Patch](https://github.com/alibaba/celeborn/blob/main/assets/spark-patch/RSS_RDA_spark3.patch).

**Comment by AngersZhuuuu** [comment, marked as answer]
在动态分配 和未开启external shuffle service的的场景下， 如果map task 所在的机器lost 是需要重跑map task的，我们开启了RSS 的时候是关闭了ESS的，所以需要做一些spark 端的修改去支持RSS的场景， 避免不必要的计算
，

**Reply by fuhaiq** [reply]
谢谢回复
